test/jdk/java/awt/print/PrinterJob/EmptyFill.java

```diff
@@ -66,11 +66,10 @@ public static void main(String arg[]) throws Exception {
        }
        ByteArrayOutputStream baos = new ByteArrayOutputStream(4096);
        StreamPrintService svc = spfs[0].getPrintService(baos);
-
-       PrinterJob pj = PrinterJob.getPrinterJob();
        if (svc == null) {
            return;
        }
+       PrinterJob pj = PrinterJob.getPrinterJob();
```

Comment:
It looks good, however, I'd move the blank so that the code groups logically:

```java
       StreamPrintService svc = spfs[0].getPrintService(baos);
       if (svc == null) {
           return;
       }

       PrinterJob pj = PrinterJob.getPrinterJob();
```

Perhaps, the test should fail if `svc` happens to be `null`, after all it's quite unexpected if it's `null`, isn't it?

Reply:
yes, it should always be possible to create the Postscript Stream PrintService - added throw new RTE.